### ISSUE TITLE
Does this open source heavydb also have a 32-gigabyte memory limit and one gpu limit?

### ISSUE THREAD

**Issue by jieguolove** [heavyai/heavydb#797]
Does this open source heavydb also have a 32-gigabyte memory limit and one gpu limit?
Is the free version the same as this open source limitation?
look this[https://www.heavy.ai/pricing](url)

**Comment by cdessanti** [comment]
Nope, the OS edition of the database can use all the GPUs and all the memory available in a single system.
It lacks features like heavy connect, backend rendering and Immerse.

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
From: jieguolove ***@***.***>
Sent: Tuesday, September 5, 2023 8:18:46 AM
To: heavyai/heavydb ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [heavyai/heavydb] Does this open source heavydb also have a 32-gigabyte memory limit and one gpu limit? (Issue #797)


Does this open source heavydb also have a 32-gigabyte memory limit and one gpu limit?

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
